### PR TITLE
hw4 for check

### PR DESCRIPTION
**Pull request by demony**
Here we have big difference in classes structure against homework_2 and 3 homework_3. 

I change it because i was getting getting the error while tried to use listeners for making picture:
"There was an error in the forked process
invalid session id"

### CODE REVIEW

File: src/test/java/com/epam/tc/hw4/pageobjects/BaseElements.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    public String getTitle() {
+        String title = webDriver.getTitle();
+        return title;
+    }
+
+    public String getCurrentUrl() {
+        return webDriver.getCurrentUrl();
+    }
```
Comment (DmitryKhodakovsky) [inline src/test/java/com/epam/tc/hw4/pageobjects/BaseElements.java]: This methods related to the `BasePage` but not for the `BaseElement`

File: src/test/java/com/epam/tc/hw4/pageobjects/BaseElements.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    public boolean closeBrowser() {
```
Comment (DmitryKhodakovsky) [inline src/test/java/com/epam/tc/hw4/pageobjects/BaseElements.java]: Browser should be closed only on test level

**Review by DmitryKhodakovsky — CHANGES_REQUESTED** [review]
Please resolve comments

File: src/test/java/com/epam/tc/hw4/pageobjects/BaseElements.java (hunk 1 above)
Comment (demony) [reply src/test/java/com/epam/tc/hw4/pageobjects/BaseElements.java]: Fixed.

**Pull request merged**